docs/cloud/execution/fargate_task_environment.md

```diff
@@ -0,0 +1,178 @@
+
+_For more information on the Fargate Task Environment visit the relevant [API documentation](/api/unreleased/environments/execution.html#fargatetaskenvironment)._
+
+## Process
+
+#### Initialization
+
+The `FargateTaskEnvironment` has two groups of keyword arguments: boto3-related arguments and task-related arguments. All of this configuration revolves around how the [boto3]() library communicates with AWS. The design of this Environment is meant to be open to all access methodologies for AWS instead of adhering to a single mode of authentication.
+
+This Environment accepts similar arguments to how boto3 authenticates with AWS: `aws_access_key_id`, `aws_secret_access_key`, `aws_session_token`, and `region_name`. These arguments are directly passed to the [boto3 client]() which means you should initialize this Environment in that same way you would normally use boto3.
+
+After specifying credentials for the boto3 client the other group of kwargs falls into the category of any arguments you would pass into boto3 for [registering](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.register_task_definition) a task definition and [running](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.run_task) that task.
+
+Accepted kwargs for [`register_task_definition`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.register_task_definition):
+
+```
+family                      string
+taskRoleArn                 string
+executionRoleArn            string
+networkMode                 string
+containerDefinitions        list
+volumes                     list
+placementConstraints        list
+requiresCompatibilities     list
+cpu                         string
+memory                      string
+tags                        list
+pidMode                     string
+ipcMode                     string
+proxyConfiguration          dict
+inferenceAccelerators       list
+```
+
+Accepted kwargs for [`run_task`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.run_task):
+
+```
+cluster                     string
+taskDefinition              string
+count                       integer
+startedBy                   string
+group                       string
+placementConstraints        list
+placementStrategy           list
+platformVersion             string
+networkConfiguration        dict
+tags                        list
+enableECSManagedTags        boolean
+propagateTags               string
+```
+
+All of these kwargs will be loaded and stored upon initialization of the Environment. It will never be sent to Prefect Cloud and will only exist inside your Flow's Docker storage.
```

Comment:
```suggestion
All of these kwargs will be loaded and stored upon initialization of the Environment. It will _never be sent to Prefect Cloud_ and will only exist inside your Flow's Docker storage object.
```